configs/config.json.example

```diff
@@ -147,6 +147,16 @@
       "max_circle_size": 50,
       "cache_recent_forts": true
     },
+    "pokemon_bag": {
+      "// if 'show_at_start' is true, it will log all the pokemons in the bag (not eggs) at bot start": {},
+      "show_at_start": true,
```

Comment:
This can be useful, but will print a great deal of spam if unwanted. Please set default value in all configs as "show_at_start": False,